CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+`arize-phoenix` is an open source project that is under active development that is being used by Arize AI customers as well as the larger AI community. This document aims to make it easy for anyone to contribute to the project.
+
+## Code of Conduct
+
+This project has adopted the [Contributor Covenant Code of Conduct](./CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code and actions that violate it will not be tolerated.
+
+## Branch Organization
+
+Submit all code changes to `main` and documentation changes to the `docs` branch.
+
+Code that lands in `main` must be compatible with the latest stable release. It may contain additional features but no breaking changes. We should be able to release a new minor version from the tip of `main` at any time.
+
+## Bugs
+
+We use GitHub issues to track bugs. We keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn’t already exist.
+
+## Pull Requests
+
+The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation.
+
+Before submitting a pull request, please make sure the following is done. We expect code authors to test their pull requests well-enough that they work correctly by the time they get to code review. (Note that `npm` commands below are run under the `./app` directory.)
+
+-   Fork the repository and create your branch from `main`.
+-   Follow the [development guide](./DEVELOPMENT.md) to setup your local environment.
+-   If you've fixed a bug or added code that should be tested, add tests!
+-   Ensure test suite pass. (`hatch run tests` and `npm run test` for app changes)
+-   Make sure your code is formatted with `hatch run style:fix` and `npm run prettier` for app changes.
+-   Make sure to your code lints with `npm run lint` for app changes.
+-   Run type checking with `hatch run type:check` and `npm run typecheck` for app changes.
+
+### Pull Request (PR) Descriptions
+
+A PR description is a public record of what change is being made and why it was made. It will become a permanent part of our version control history, and will possibly be read by many people other than your reviewers over the years. Follow the following guidelines when writing a PR description:
+
+-   Title: The tile must conform to [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) format and must sufficiently describe the change. Since PR titles are used to form release notes, titles with generic or non-descriptive content (“Fix build.”, “Add patch.”) are not allowed.
+-   Issue: The first line of the description should contain a reference to the issue that the PR is solving. For example, `fixes #1234` or `resolves #1234`. While this is not required for urgent fixes, it is required for all other PRs so that the issue is clearly tracked and triaged by a core team member.
```

Comment:
Does this need to be part of the PR description itself? I usually link the issue in the `Development` section, but don't leave a link in the PR description.

Reply:
yeah it's the easiest way to do it and will probably be added to the PR template long term.